frame/contracts/src/lib.rs

```diff
@@ -129,6 +124,12 @@ use sp_core::{crypto::UncheckedFrom, Bytes};
 use sp_runtime::traits::{Convert, Hash, Saturating, StaticLookup};
 use sp_std::{fmt::Debug, marker::PhantomData, prelude::*};
 
+pub use crate::{
+	exec::{FixSizedKey, Frame, VarSizedKey},
```

Comment:
e.g. `node-executor` basic test uses it [here](https://github.com/paritytech/substrate/blob/ag-transparent-hashing/bin/node/executor/tests/basic.rs#L769)

Reply:
But that is not necessary? It would be best if you didn't introduce new users of deprecated types. Just use a `VarSizedKey` there. Then the fixed key can be private. Also, we should export `VarSizedKey` just as `StorageKey`. Let's hide the fixed-size key or the fact that there are multiple key types as well as we can. It is just a detail for pre-existing contracts.